### DISCUSSION TITLE
WinUI and Windows App SDK at Ignite 2024!

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/microsoft-ui-xaml#10186] [General]
Hi everyone,

As we kick off Ignite 2024 this year, we've lined up several sessions for WinUI and Windows App SDK that we think you'll love! Although not as focused on native Windows development as //Build, Ignite is still a great place to learn about our technologies and see what's coming next, if you know where to look!

- See an overview of Windows announcements at Ignite on the Windows Blog: [Microsoft Ignite 2024: Embracing the future of Windows at work](https://blogs.windows.com/windowsexperience/2024/11/19/microsoft-ignite-2024-embracing-the-future-of-windows-at-work/)
- Returning from //Build by popular demand and with freshly updated content, learn how to choose the right tech for your Win32 app development on Windows: [Navigating Win32 App Development with WinUI and WPF](https://aka.ms/igniteBRK305)
  - This session is scheduled for Wednesday, November 20th at 8:30 AM CST (2:30 PM UTC)
- Get a deeper look at the new Windows Copilot Runtime APIs, coming soon in the Windows App SDK, and how they fit into enterprise apps: [Integrate AI in enterprise apps using Windows Copilot Runtime APIs powered by on-device ML models](https://aka.ms/igniteBRK303)
  - This session is scheduled for Thursday, November 21st at 9:45 AM CST (3:45 PM UTC)
 - See how one of the biggest app suites in the world, Office, takes advantage of the Windows App SDK and ReactNative: [Modernize your apps by learning how Office leverages Windows App SDK](https://aka.ms/igniteBRK304)
    - This session is scheduled for Thursday, November 21st at 1:15 PM CST (7:15 PM UTC)

We hope you have a great time at Ignite and look forward to seeing you there!

**Comment by ghost1372** [comment]
When will v1.7-experimental 1 be released?  Last week @michael-hawker announced on YouTube that next week (this week) v1.7-exp1 will be released..

**Reply by niels9001** [reply]
It will be released today. Stay tuned!

**Reply by DarranRowe** [reply]
It seems like it is on NuGet.
